packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -262,7 +281,12 @@ class RenderEditable extends RenderBox {
       newOffset = _handleShift(rightArrow, leftArrow, shift, newOffset);
 
       _extentOffset = newOffset;
+    } else if (ctrl && (xKey || vKey || cKey || aKey)) {
+      // Do not await this method
```

Comment:
you can't await it, this calling method isn't `async`. The important thing here is that `_handleShortcuts` depends on being started in the same stack invocation as the `_handleKeyEvent` method, synchronously with the keyboard event being received.